aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -368,6 +373,16 @@ module aptos_framework::stake {
         borrow_global<StakePool>(pool_address).operator_address
     }
 
+    #[view]
+    /// Return the operator of the validator at `pool_address`.
+    public fun get_beneficiary_for_operator(pool_address: address): address acquires StakePool, Beneficiary {
+        assert_stake_pool_exists(pool_address);
+        if(exists<Beneficiary>(pool_address)) {
```

Comment:
nit: space after if

Reply:
This code is removed from the stake module.